src/screens/Asset/Asset.js

```diff
@@ -230,10 +231,10 @@ class AssetScreen extends React.Component<Props, State> {
   };
 
   handleScrollWrapperEndDrag = e => {
-    const { fetchAssetTransactions, history } = this.props;
-    const {
-      assetData: { token },
-    } = this.props.navigation.state.params;
+    const { fetchAssetTransactions, history, activeAccount } = this.props;
+    if (!activeAccount || checkIfSmartWalletAccount(activeAccount)) return;
```

Comment:
The prop `activeAccount` is not marked as optional, I'd avoid the `!activeAccount` check (or make it optional if makes more sense).

Reply:
Added change to checkIfSmartWalletAccount just to check if passed account object is valid.